mmv1/provider/terraform/custom_code.rb

```diff
@@ -89,6 +89,10 @@ class CustomCode < Api::Object
       # should be used for resources that don't have normal creation
       # semantics that cannot be supported well by other MM features.
       attr_reader :custom_create
+      # This code replaces the entire contents of the Read call. It
+      # should be used for resources that don't have normal read
+      # semantics that cannot be supported well by other MM features.
+      attr_reader :custom_read
```

Comment:
Instead of adding `custom_read` and using that to implement an exclude, could we add an `exclude_read` method instead? That'll be more clear, and mean we don't add the `custom_read` method. While custom_x methods are _fine_, needing one for read in particular points to particularly complex resources that we may want to handwrite.